components/AppBar.tsx

```diff
@@ -51,80 +53,101 @@ export function AppBar() {
-            height={40}
-          />
-          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
-            NetZeroPaths
-            <Typography variant="caption" sx={{ display: 'block' }}>
-              Powered by Kausal Open Source
-            </Typography>
-          </Typography>
-        </Stack>
+  const handleOpenSupportModal = () => {
+    setSupportModalOpen(true);
+  };
 
-        <div>
-          <Button
-            size="small"
-            variant="text"
-            aria-controls="appbar-auth-menu"
-            aria-haspopup="true"
-            onClick={handleOpenAuthMenu}
-            color="inherit"
-            endIcon={<PersonCircle size={22} />}
-          >
-            {isAuthenticated
-              ? getUserDisplay(session.data)
-              : 'Create Your City Plan'}
-          </Button>
+  const handleCloseSupportModal = () => {
+    setSupportModalOpen(false);
+  };
 
-          <Menu
-            id="appbar-auth-menu"
-            anchorEl={menuAnchorEl}
-            anchorOrigin={{
-              vertical: 'top',
-              horizontal: 'right',
-            }}
-            keepMounted
-            transformOrigin={{
-              vertical: 'top',
-              horizontal: 'right',
+  return (
+    <>
+      <MuiAppBar position="sticky" sx={APP_BAR_STYLES}>
+        <Toolbar>
+          <Stack
+            component={Link}
+            href="/"
+            direction="row"
+            spacing={4}
+            sx={{
+              color: 'inherit',
+              textDecoration: 'none',
+              alignItems: 'center',
+              flexGrow: 1,
             }}
-            open={!!menuAnchorEl}
-            onClose={handleCloseAuthMenu}
           >
-            {!isAuthenticated ? (
-              [
-                <MenuItem key="sign-up" onClick={handleSignUp}>
-                  Sign up
-                </MenuItem>,
-                <MenuItem key="log-in" onClick={handleLogIn}>
-                  Log in
-                </MenuItem>,
-              ]
-            ) : (
-              <MenuItem onClick={handleSignOut}>Sign out</MenuItem>
-            )}
-          </Menu>
-        </div>
-      </Toolbar>
-    </MuiAppBar>
+            <Image
+              src="/images/net-zero-cities-logo.png"
+              alt="Net Zero Cities"
+              width={80}
+              height={40}
+            />
+            <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+              NetZeroPaths
+              <Typography variant="caption" sx={{ display: 'block' }}>
+                Powered by Kausal Open Source
+              </Typography>
+            </Typography>
+          </Stack>
+
+          <div>
+            <Stack direction="row" spacing={2}>
+              <Button
+                size="small"
+                variant="text"
+                color="inherit"
+                onClick={handleOpenSupportModal}
+                endIcon={<InfoCircle size={22} />}
```

Comment:
Let's go with the `QuestionCircle` icon instead, this bootstrap icon doesn't quite fit

https://icons.getbootstrap.com/icons/question-circle/